### ISSUE TITLE
Create sample page for SVG naming support

### ISSUE THREAD

**Issue by smhigley** [fizzstudio/svg-aria-test#2]
support for SVG (not graphics-document roles) and aria-label / aria-labelledby

**Comment by distantorigin** [comment]
See #14 for detailed versions of the software used during testing.

###General observations
- All SVGs with an explicitly set tabindex are tabbable to with keyboard or swipable on mobile.
- Voice Control on iOS can activate items using the clickable class.

### JAWS + Chrome
- SVG with aria-label (test 1) - both are identified as graphics and announced as "Red square graphic" and "Blue circle graphic"
- It is not conveyed that the second SVG (blue circle) is clickable, although pressing enter with JAWS enabled works reliably.
- The following tests are not exposed unless tabbed to by the user:
	- Shapes with aria-label
	- Group with aria-label
	- Path with aria-label
	- Layered svg, group, and path names

### JAWS + Firefox
- All SVGs among all tests are announced as static text without a role.
- There are no spaces or delimiters in between each element, thus text runs together when rendered by JAWS. An example of the "Shapes with aria-label" test:
```
Circles:
red circleblue circle
Ellipses:
red ellipseblue ellipse
Lines:
red lineblue line
Polylines:
red polylineblue polyline
Polygons:
red polygonblue polygon
Rectangles:
red rectangleblue rectangle
```

### NVDA + Chrome
- SVG with aria-label: Both shapes are identified appropriately by their aria-label and are announced as a graphic (i.e. "Graphic blue circle"). The second SVG is not identified as clickable.
- Shapes with aria-label: All shapes are identified as graphics and their aria-labels are announced. Clickable items are not announced as clickable.
- Group with aria-label: All shapes are identified as graphics and their aria-labels are announced. Clickable items are not announced as clickable.
- Path with aria-label: Similarly to above, all paths are identified as graphics and their aria-labels are announced (i.e. "Graphic Red Triangle"). No information, such as coordinates it occupies or relative geometry about the path is conveyed.
- Layered svg, group, and path names: Using arrow keys, the group name and various elements are announced as "Shapes graphic shapes red blue" without pause. Tabbing announces "Blue shapes graphic" but there is no graphic that reads "blue shapes" when using the arrow keys, which can be confusing. No other information is conveyed about the relation between the paths. The fact that this is a group of SVGs is also not conveyed.

### NVDA + Firefox
- All tests fail. None of the SVG objects are announced in any tests. Pressing the tab key results in NVDA announcing "Blank". Pressing the G key to navigate to the next graphic denotes that there are no graphics on the page.

### Safari + VoiceOver
- SVG with aria-label: VoiceOver fails to identify aria-labels. Both elements are announced as "image" only.
- Shapes with aria-label: All shapes are identified as images and aria-labels are announced. Clickable items are not announced as clickable.
- Group with aria-label: Group names are announced as static text without any roles.
- Path with aria-label: Similarly to above, all paths are identified as images and their aria-labels are announced (i.e. "blue triangle image"). No information, such as coordinates it occupies or relative geometry about the path is conveyed.
- Layered svg, group, and path names: The only text exposed is the group name, "Triangles." It is not identified as a group, however, and for all intents and purposes appears as static text. Paths and shapes are not read by VoiceOver. Double-tapping the group name with VoiceOver enabled does nothing.
